backend/blueprints/spa_api/service_layers/replay/replay_positions.py

```diff
@@ -0,0 +1,78 @@
+import gzip
+import os
+
+import pandas as pd
+
+from typing import List, cast
+from flask import current_app
+
+from carball.analysis.utils import proto_manager, pandas_manager
+from backend.database.objects import Game, PlayerGame
+from .replay_player import ReplayPlayer
+from ..utils import sort_player_games_by_team_then_id
+from ...errors.errors import ReplayNotFound, ErrorOpeningGame
+
+
+class ReplayPositions:
+    def __init__(self, id_: str,
+                 ball, players, colors, names,
+                 frames):
+        self.id = id_
+
+        self.ball = ball
```

Comment:
not bad, but I'd like to be able to see what types the attributes are from this `__init__`. ie. what the .ball is, or .colors is - .colors is a list of strings? list of booleans?

What I've done so far is type the parameters to the `__init__`.